### ISSUE TITLE
b54: Low contrast illegible text in the new pill

### ISSUE THREAD

**Issue by bradtchapman** [BillGoldsmith/iOS-bugfest#114]
"Support RP" is illegible in the middle of the pill - white text against yellow and bright green fails the WCAG accessibility contrast test. 

Use a thin black outline around the text, or darken the colors in the pill, or redesign it. 

![IMG_2109](https://user-images.githubusercontent.com/32527656/232119896-fb88346c-b6ea-44ce-a780-9add8db2f059.jpeg)

**Comment by RPNate1** [comment]
We'll get this inline with whats on the website. even that isn't ideal but...

**Comment by bradtchapman** [comment]
@RPNate1 : no problem; I just noticed it because it's a change in b54.

**Comment by RPNate1** [comment]
Fixed on b55. (same as website, not ideal contrast, but works)
